package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -167,6 +167,11 @@ local function configure_mesh(config, radio, index, suffix, disabled)
 		macfilter = macfilter,
 		maclist = maclist,
 	})
+
+	if config.psk(false) then:
+		uci:set('wireless', 'wifi-iface', name, 'encryption', 'psk2+ccmp')
```

Comment:
psk2+ccmp means WPA2. Your pull request description is about WPA3. WPA3 would be 'sae' according to https://openwrt.org/docs/guide-user/network/wifi/basic#wpa_modes